### DISCUSSION TITLE
What is the difference between bbox ap and segm ap?

### DISCUSSION THREAD

**Discussion by vansin** [open-mmlab/mmeval#49] [Announcements]

**Comment by ice-tong** [comment]
Hi @vansin ,  the 'bbox' and 'segm' ap in [COCODetectionMetric](https://github.com/open-mmlab/mmeval/blob/e2f6dbf160d41adfa185caed211f2ac50747f3ba/mmeval/metrics/coco_detection.py#L33) follows the definitions in https://cocodataset.org/#detection-eval

> There are two object detection tasks for COCO dataset: using either bounding box output or object segmentation output (the latter is also known as **instance segmentation**). 

> The evaluation metrics for detection with bounding boxes and segmentation masks are identical in all respects except for the **IoU computation** (which is performed over boxes or masks, respectively).
